api/nodes/serializers.py

```diff
@@ -212,6 +206,7 @@ def profile_image_url(self, user):
 
     class Meta:
         type_ = 'contributors'
+        target_type_ = 'users'
```

Comment:
/v2/nodes/{node_id}/contributors/ is a relationship URL.  It seems to me that the only possible relationship you could create by posting to this URL is the one between the node and a user, called a contributor.  The URL shows all of the relationships between this node and other users.  Likewise for /v2/nodes/{node_id}/node_links.   Therefore, the way we have it now, the only target_type_ would be 'users'.

It would be different if POSTing to /v2/nodes/ we could create a node and simultaneously create a relationship between a user and a node link.  /v2/nodes/ is not a relationship URL, it is a URL describing a collection of nodes.  So there we might be creating multiple types of relationships simultaneously.